internal/consoleplugin/config.go

```diff
@@ -0,0 +1,223 @@
+package consoleplugin
+
+import (
+	"fmt"
+
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Config holds configuration for the logging console plugin.
+// Construct with NewConfig.
+type Config struct {
+	CreatedBy string
+	Name      string
+	Namespace string
+	Image     string
+	LokiName  string
+	LokiPort  int32
+
+	nginxPort, defaultMode, replicas int32
+}
+
+// NewConfig returns a config with default settings.
+// Modify exported fields to change settings.
+func NewConfig(namespace, createdBy string) *Config {
+	return &Config{
+		CreatedBy: createdBy,
+		Name:      "logging-view-plugin",
+		Namespace: namespace,
+		Image:     "quay.io/gbernal/logging-view-plugin:latest", // FIXME
```

Comment:
There is a utility for fetching this.  search `RELATED_` as this also requires manifest changes